GitHubCodespaces.ps1

```diff
@@ -898,6 +1074,241 @@ function Wait-GitHubCodespaceAction
+filter Add-GitHubCodespaceUser
+{
+    <#
+    .SYNOPSIS
+        Add user(s) to Codespaces billing for an organization.
+
+    .DESCRIPTION
+        Add user(s) to Codespaces billing for an organization.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OrganizationName
+        Name of the Organization.
+
+    .PARAMETER UserName
+        The handle for the GitHub user account(s).
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .EXAMPLE
+        Add-GitHubCodespaceUser -OrganizationName microsoft -UserName HowardWolosky
+
+    .OUTPUTS
+        [PSObject[]]
+
+    .NOTES
+        To use this endpoint, the billing settings for the organization must be set to selected_members.
+        For information on how to change this setting please see [these docs](https://docs.github.com/rest/codespaces/organizations#manage-access-control-for-organization-codespaces)
+
+    .NOTES
+        You must authenticate using an access token with the admin:org scope to use this endpoint.
+
+    .LINK
+        https://docs.github.com/en/rest/codespaces/organizations?apiVersion=2022-11-28#add-users-to-codespaces-billing-for-an-organization
+    #>
+    [CmdletBinding(SupportsShouldProcess)]
+    param (
+        [Parameter(Mandatory)]
+        [string] $OrganizationName,
+
+        [Parameter(
+            Mandatory,
+            ValueFromPipeline,
+            ValueFromPipelineByPropertyName)]
+        [string[]] $UserName,
+
+        [string] $AccessToken
+    )
+
+    Write-InvocationLog
+
+    $hashBody = @{
+        selected_usernames = @($UserName)
+    }
+
+    $params = @{
+        UriFragment = "orgs/$OrganizationName/codespaces/access/selected_users"
+        Body = ConvertTo-Json -InputObject $hashBody
+        Method = 'Post'
+        Description = 'Add users to GitHub codespace billing'
+        AccessToken = $AccessToken
+        TelemetryEventName = $MyInvocation.MyCommand.Name
+    }
+
+    if (-not $PSCmdlet.ShouldProcess(($UserName -join ','), 'Add Codespace Users'))
+    {
+        return
+    }
+    Invoke-GHRestMethod @params
+}
+
+filter Remove-GitHubCodespaceUser
+{
+    <#
+    .SYNOPSIS
+        Remove user(s) from Codespaces billing for an organization.
+
+    .DESCRIPTION
+        Remove user(s) from Codespaces billing for an organization.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OrganizationName
+        Name of the Organization.
+
+    .PARAMETER UserName
+        The handle for the GitHub user account(s).
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .EXAMPLE
```

Comment:
Can you list some relevant `.INPUTS` here as well?

Reply:
GitHub.User is the only relevant input I can imagine.  cba3aa480517e2c9f8bff6c46fd37b5011d07225